openbas-api/src/main/java/io/openbas/rest/settings/PlatformSettingsApi.java

```diff
@@ -0,0 +1,279 @@
+package io.openbas.rest.settings;
+
+import io.openbas.config.OpenBASConfig;
+import io.openbas.config.OpenBASPrincipal;
+import io.openbas.database.model.Setting;
+import io.openbas.database.model.Theme;
+import io.openbas.database.repository.SettingRepository;
+import io.openbas.injects.opencti.config.OpenCTIConfig;
+import io.openbas.rest.helper.RestBehavior;
+import io.openbas.rest.settings.form.ThemeInput;
+import io.openbas.rest.settings.response.OAuthProvider;
+import io.openbas.rest.settings.response.PlatformSettings;
+import jakarta.annotation.Resource;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.oauth2.client.OAuth2ClientProperties;
+import org.springframework.boot.autoconfigure.security.saml2.Saml2RelyingPartyProperties;
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.env.Environment;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static io.openbas.config.SessionHelper.currentUser;
+import static io.openbas.database.model.Setting.SETTING_KEYS.*;
+import static io.openbas.database.model.User.ROLE_ADMIN;
+import static io.openbas.helper.StreamHelper.fromIterable;
+import static java.util.Optional.ofNullable;
+
+
+@RestController
+public class PlatformSettingsApi extends RestBehavior {
```

Comment:
There are some duplicate code from SettingsApi file
This two files (SettingsApi & PlatformSettingsApi) need to be merge